draft-ietf-quic-transport.md

```diff
@@ -205,28 +205,28 @@ Out-of-order packet:
 
 Endpoint:
 
-: An entity that can participate in a QUIC connection by generating,
-  receiving, and processing QUIC packets. There are only two types of endpoint
-  in QUIC: client and server.
+: An entity that can participate in a QUIC connection by generating, receiving,
+  and processing QUIC packets. There are only two types of endpoint in QUIC:
+  client and server.
 
 Client:
 
 : The endpoint initiating a QUIC connection.
 
 Server:
 
-: The endpoint accepting incoming QUIC connections.
+: The endpoint accepting an incoming QUIC connection.
 
 Address:
 
-: When used without qualification, the tuple of IP version, IP address, UDP
-  protocol, and UDP port number that represents one end of a network path.
+: When used without qualification, the tuple of IP version, IP address, and UDP
+  port number that represents one end of a network path.
 
 Connection ID:
 
 : An opaque identifier that is used to identify a QUIC connection at an
-  endpoint.  Each endpoint sets a value for its peer to include in packets sent
-  towards the endpoint.
+  endpoint.  Each endpoint sets one or more values for its peer to include in
+  packets sent towards the endpoint.
```

Comment:
I don't feel great about this ... it is slightly better than the old text but it's still unclear exactly what is meant. How about "This identifier is set by an endpoint for its peer ..."

Reply:
That's workable, but the point of my edit is not to imply the CID is 1:1 with the connection itself.  Perhaps we use similar words to the version change being proposed, "Opaque bytes that permit the recipient to associate a packet with a connection.  Each endpoint can supply one or more sequences of bytes for its peer to use in sending packets."?

Reply:
I think that we should be saying "selects one or more connection IDs".  I might have misread the suggestion here like Jana and read it to mean that a connection ID is "one or more values".